src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,149 @@
+
+At the root of the project, you'll find [`step1/`](https://github.com/delapuente/aframe-interactive-training/tree/master/step1), [`step2/`](https://github.com/delapuente/aframe-interactive-training/tree/master/step2) and [`step3/`](https://github.com/delapuente/aframe-interactive-training/tree/master/step3) with the **finished HTML** for each step. Look at the same folders inside the [`js/`](https://github.com/delapuente/aframe-interactive-training/tree/master/js) directory to browse the JavaScript code.
+
+Open the browser and enter the URL `localhost:8080` to find the index of steps. Append `step1` at the end of the URL path and click on the VR button on the bottom-right corner of the display to experience the project after step 1 is completed (or select step 1 in the list).
+
+The project is based on the [A-Frame SPA skeleton](https://github.com/delapuente/aframe-spa-skeleton); you can use this to quickly set up the barebones of a virtual reality single-page application with auto reload and publishing features. Read the [a-frame-skeleton README file](https://github.com/delapuente/aframe-spa-skeleton#a-frame-spa-skeleton) for further information.
+
+## Planning the demo
+
+The app’s initial aim was to provide a simulation of the process of backing up the contents of a chip from an in-car key reader device, which includes loosening the solder and removing the chip from a circuit board. As a reference, I was given a YouTube video showing the [complete in-real-life procedure](https://youtu.be/qFGX3AHVLqA?t=90). The video comprises several steps involving around eight different tools. For this series, I'm going to focus on the specific step of separating the chip from the circuit board. See a detailed sequence here:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/tHP2kX6aAZM?rel=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+The required steps are as follow:
+
+  1. With a hot air gun, soften the soldering around the chip.
+  2. With the help of a suction pad, separate the chip from the circuit board.
+  3. Carefully place the chip on the table, to avoid ruining the chip pins.
+
+I chose to implement one step at a time without realistic models in the beginning. Once I am able to recreate the complete process, I can think about adding different levels of detail to the demo.
+
+But before providing specific simulations and behaviors, we need some basic hand-based interaction.
+
+## Setting the scene up
+
+Let’s start looking at the code. The first thing I needed was a scene containing all the elements I wanted to work with: the air gun, the suction pad, the chip, the soldering around the chip, and the circuit board. Of course I needed a representation of my own hands and some environmental elements such as a sky, a floor and of course a table!
```

Comment:
We can maybe omit `Let’s start looking at the code` and go straight to `The first thing we need is...`